docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -68,7 +68,17 @@ This decreases the number of segments, which means less metadata is kept in
 heap memory.
 
 Every mapped field also carries some overhead in terms of memory usage and disk
-space. By default {es} will automatically create a mapping for every field in
+space. While the exact amount resource usage of each additional field depends
+on the type of field, an additional 1024B in memory use  for each additional
+mapped field on data nodes is a good approximation in most cases.
+As a result, the more mapped fields you have per index, the fewer
+indices and shards each individual data node will be able to hold,
+all other things being equal. In addition to the per field overhead you should
+at least have an additional 512MB of heap available per data node.
```

Comment:
I think this moves a proportion of our customers from inside to outside our recommendations, which I think is not the purpose here. I suggest to rephrase to something like:

In addition to the per field overhead you should have heap available for the base Elasticsearch heap usage as well as the heap usage of the workload. For a small workload, 512MB of heap in addition to the per field overhead suffice in many cases. For tight workloads, it is possible to go even lower.

Reply:
++ reworded to your suggestion